### PR TITLE
Additions of ecoinvent and IDEA mapping files to the FEDEFL

### PR DESCRIPTION
**Pull request by ashleyedelen**
These are drafts of the mapping files for IDEA and ecoinvent to the FEDEFL. They have undergone a preliminary internal review by ERG staff, but further review is recommended before finalizing.

### CODE REVIEW

**Review by WesIngwersen — CHANGES_REQUESTED** [review]
See my comments requesting changes to these two files in the commits of your fork

**Comment by ashleyedelen** [comment]
I uploaded the correct files. These files should include the correct naming.

the {} are from the IDEA source and are a copy of the original source

I believe the other issues have been corrected with the correct upload of the files.

**Comment by WesIngwersen** [comment]
@ashleyedelen Please see the posted version of IDEAv2.2 in the main repo - the {} are not there
https://raw.githubusercontent.com/UNEP-Economy-Division/GLAD-ElementaryFlowResources/master/flowlists/IDEAv2.2.csv
And please confirm again that the flows you used match those in this list. thanks

**Comment by hottleta** [comment]
I added the split flows to the bottom of the IDEA mapping for additional consideration. Depending on what we want to do, we can go ahead and map those individually.

**Review by WesIngwersen — CHANGES_REQUESTED** [review]
@hottleta Please go ahead and propose actual mappings for those split flows rather than leaving the [potential split flows] in the file

File: mapping/IDEAv2.2toFEDEFLv1.0.3.csv [hunk 1]
```diff
@@ -0,0 +1,1022 @@
...
+IDEAv2.2,barren land,60a31ee1-b87b-40e9-8afb-b5ad60994c06,Resources/land use/land occupation,m2year,=,1,Land use,879bee81-f5fc-351e-9d0c-9cd219d40cd6,resource/ground/terrestrial/barren land,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,forest,fcd5e16b-e4c5-4c60-b045-042d30abd42d,Resources/land use/land occupation,m2year,=,1,Land use,1147e66c-db3e-303c-b7c7-89b006d98881,resource/ground/terrestrial/forest,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,mineral extraction site,CC0F5788-8059-4253-8498-D7B8D3CD3CD6,Resources/land use/land occupation,m2year,=,1,Land use,4b92c75e-e528-3f23-bc67-e92977682eb4,resource/ground/human-dominated/industrial,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,dump yard,dd8c94cc-b5c2-4736-9dfc-668bce100891,Resources/land use/land occupation,m2year,=,1,Land use,4b92c75e-e528-3f23-bc67-e92977682eb4,resource/ground/human-dominated/industrial,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,Iron manganese ore,1404b845-05d0-4d16-94f5-999775446c55,Resources/ground/Non-renewable material resources,kg,>,0.5,Iron ore,6fca43a7-13fb-315d-a184-8af7d40ff1d3,resource/ground,kg,Hottle,,8/28/2020
```
Comment (WesIngwersen) [inline mapping/IDEAv2.2toFEDEFLv1.0.3.csv:1015]: @kshobatake Please see mappings in this line forward where we split 1 IDEA flow across two FEDEFL flows. The conversion factors are also split in equal proportion to add to 1. If you have insight into better % composition data of the members of those groups in the IDEA flows, that could be used to improve the conversion factors

File: mapping/ecoinventEFv3.6toFEDEFLv1.0.3.csv [hunk 2]
```diff
@@ -0,0 +1,4103 @@
+SourceListName,SourceFlowName,SourceFlowUUID,SourceFlowContext,SourceUnit,MatchCondition,ConversionFactor,TargetFlowName,TargetFlowUUID,TargetFlowContext,TargetUnit,Mapper,Verifier,LastUpdated
+ecoinvent_v3.6,"1,4-Butanediol",38a622c6-f086-4763-a952-7c6b3b1c42ba,air/urban air close to ground,kg,=,1,"1,4-Butanediol",34d53001-84a2-3053-b887-32b891e334eb,emission/air/troposphere/urban/ground-level,kg,Edelen,Hottle,8/21/2020
```
Comment (WesIngwersen) [inline mapping/ecoinventEFv3.6toFEDEFLv1.0.3.csv]: @carl-vadenbo Please see the name used here. In the mapping files the SourceListName needs to match exactly the list name in the FlowList files, and the filenames of these files.  Please inform us of the standard name that you wish to use for this Ecoinvent flow list v3.6. thanks

**Review by WesIngwersen — CHANGES_REQUESTED** [review]
Some fields are missing from the required format ..TargetListName and the 4 Memo fields at the end
See https://github.com/UNEP-Economy-Division/GLAD-ElementaryFlowResources/blob/master/formats/FlowMapping.md
which i just put up but was on the dropbox.

File: mapping/ecoinventEFv3.6toFEDEFLv1.0.3.csv (hunk 2 above)
Comment (carl-vadenbo) [reply mapping/ecoinventEFv3.6toFEDEFLv1.0.3.csv]: @WesIngwersen Thanks for spotting this inconsistency. I haven't been able to locate where "ecoinvent_v3.6" is present, but let's go for "ecoinventEFv3.6" in any case. Please just let me know, should you need any further actions from my side to fix this, thanks!

Comment (WesIngwersen) [reply mapping/ecoinventEFv3.6toFEDEFLv1.0.3.csv]: @carl-vadenbo what about just `ecoinventv3.6` ? i think that would be more consistent with the other list names like `IDEAv2.2` . the `EF` is implicit

Comment (carl-vadenbo) [reply mapping/ecoinventEFv3.6toFEDEFLv1.0.3.csv]: @WesIngwersen sure, I'm fine with that, thanks!

**Pull request closed without merging**